examples/react-native/src/Home.js

```diff
@@ -22,14 +22,14 @@ import {
   connectSortBy,
   connectRange,
   connectCurrentRefinements,
-} from 'react-instantsearch/connectors';
-import Highlight from './components/Highlight';
-import Spinner from './components/Spinner';
+} from 'react-instantsearch-core';
```

Comment:
We should import from `react-instantsearch-native`. The `core` package is only used internally.